src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = 'Arcus.Scripting.ApiManagement.psm1'
 
 # Version number of this module.
-ModuleVersion = '#{Package.Version}#'
+ModuleVersion = '0.1.0'
```

Comment:
Replace this back to the parameterized version ;)

Reply:
Hehe.

Reply:
Done.